### DISCUSSION TITLE
Color font of date and time at home page

### DISCUSSION THREAD

**Discussion by Loumene** [Blackymas/NSPanel_HA_Blueprint#1416] [Q&A] [answered]
Hello, I would like to know if there is a way to change the font colour of the date and time on the home page.
I was able to edit the .hri file with Nextion Editor without any problems (icons and other font colours), except for the date and time on the home page where I can change the font but not the colour, it keeps the white colour despite the changes. 
Is this coded somewhere else? Or have I missed something stupidly? 
Thanks for any help.

**Comment by edwardtfn** [comment, marked as answer]
On the automation where you set your panel using the Blueprint, go all the way down and you will find the time and date color inputs:

![IMG_6651](https://github.com/Blackymas/NSPanel_HA_Blueprint/assets/94725493/c7499f11-0ad8-496c-ba8a-4afeb0a035b0)

**Reply by Loumene** [reply]
Oh I was so stupid, I've gone through the options so many times but I hadn't seen those options at the bottom ^^

Sorry for stupid question and tks for answer :)
